Sources/Classes/Helpers/Vendors/AppleUtils.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  AppleUtils.swift
+//  MetricsReporter
+//
+//  Created by Desu Sai Venkat on 23/11/23.
+//
+
+import Foundation
+
+func getOsVersion() -> String {
```

Comment:
It would have been better if we took it inside a class and made this method static.